src/main/java/com/chopshop166/chopshoplib/maps/DifferentialDriveMap.java

```diff
@@ -0,0 +1,22 @@
+package com.chopshop166.chopshoplib.maps;
+
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+
+import edu.wpi.first.wpilibj.GyroBase;
+
+/**
+ * Differential Drive Map
+ * <p>
+ * Gets the Left {@link SendableSpeedController}, gets the right
+ * {@link SendableSpeedController}, and gets {@link Gyro}
+ * 
+ * @author Jeffrey Burke
+ * @since 2020-01-20
+ */
+public interface DifferentialDriveMap {
+    public SendableSpeedController getLeft();
```

Comment:
We might be able to exterminate two avians with one pebble, and say that these should return `EncodedSpeedController` instead. Thoughts, @bot190 ?

Reply:
Also, PMD complains that anything within an interface is public by default, so has no need to explicitly say `public`. Please remove those.

Reply:
I agree that this should return `EncodedSpeedController`. We may want a helper in that class that will create one with a mock encoder.